doc/ctrlsf.txt

```diff
@@ -430,6 +430,20 @@ as its counterpart in Ag/Ack.
 >
     let g:ctrlsf_context = '-B 5 -A 3'
 <
+g:ctrlsf_compact_winsize                             *'g:ctrlsf_compact_winsize'*
+Default: 'auto'
+Available: 'auto', 'xx%', 'xx'
+Height of CtrlSF compact window. It accepts string as its value and there are
+3 types of argument:
+>
+    'auto' : half of current vim window size.
```

Comment:
This is not true. The default value is 10.

Reply:
fixed

Reply:
Sorry for misleading you.

I think a fixed value 10 (height of builtin qucikfix window) is a sensible default value for `g:ctrlsf_compact_winsize`. To me, `auto` seems unnecessary for this option, it's better to remove it from the first place.

Reply:
I did indeed set the default to 10 lines in [plugin/ctrlsf.vim](https://github.com/dyng/ctrlsf.vim/pull/302/files#diff-3ce073688d1e1b287840d5051b330c673aba328b41d93ec6cc89771255c886aeR294). =)

I had the same thought about perhaps removing `auto`, but I thought I'd keep it to match the options available for `g:ctrlsf_winsize`. If you would still like for me to remove it then I certainly can.

Reply:
Ok, it's not a big deal. Let's merge it.